packages/create-fuels/src/cli.ts

```diff
@@ -86,7 +103,7 @@ export const runScaffoldCli = async ({
 
   await mkdir(projectPath);
 
-  const templateDir = join(__dirname, '..', 'templates', templateName);
+  const templateDir = join(__dirname, '..', 'templates', templateOfChoice);
```

Comment:
Maybe a future improvement, but we could make the bootstrapping based [logic](https://github.com/FuelLabs/fuels-ts/pull/2851/files#diff-a1150dd0573a07b2a93609a4fc4fb055d41c138716e3c27515bf25cc23537787R106-R137) live in the template.

Reply:
Yes, some split of responsibility may be necessary.

Perhaps this can be better evaluated during the Vite addition.

Reply:
I would agree, tackle it when we have an example at hand.